### DISCUSSION TITLE
"GENROU (xl <= xd2) out of typical upper limit ".What this message means?

### DISCUSSION THREAD

**Discussion by m-jam20** [CURENT/andes#520] [Q&A]
Hi,

After running ieee39_full.xlsx, I gt this message:

Input file parsed in 0.0733 seconds.
System internal structure set up in 0.0384 seconds.
-> System connectivity check results:
  No islanded bus detected.
  System is interconnected.
  Each island has a slack bus correctly defined and enabled.

-> Power flow calculation
           Numba: Off
   Sparse solver: KLU
 Solution method: NR method
Power flow initialized in 0.0034 seconds.
0: |F(x)| = 7.561234472
1: |F(x)| = 2.916445037
2: |F(x)| = 0.1312866396
3: |F(x)| = 0.0003487368405
4: |F(x)| = 4.215688243e-09
Converged in 5 iterations in 0.0071 seconds.
Report saved to "ieee39_full_out.txt" in 0.0083 seconds.
GENROU (xl <= xd2) out of typical upper limit.

   idx     | values | limit
-----------+--------+------
 GENROU_1  | 0.012  | 0.001
 GENROU_2  | 0.042  | 0.036
 GENROU_3  | 0.036  | 0.003
 GENROU_4  | 0.025  | 0.001
 GENROU_5  | 0.050  | 0.001
 GENROU_7  | 0.031  | 0.002
 GENROU_8  | 0.029  | 0.006
 GENROU_9  | 0.018  | 0.001
 GENROU_10 | 0.003  | 0.000

Initialization for dynamics completed in 0.0415 seconds.
Initialization was successful.

**Comment by cuihantao** [comment]
This is a warning that will not impact the numerical results.

In a typical generator model, the leakage reactance `xl` is smaller than the subtransient reactance `xd''`. There may be parameter issues needing correction for this test case.

**Reply by m-jam20** [reply]
Thanks. 

I have used ieee39_full.xlsx provided in "https://github.com/CURENT/andes/tree/master/andes/cases/ieee39". 

You mean there might be parameter issues in this xlsx file?

**Reply by m-jam20** [reply]
I also get this when I want to run the model with 0.8 of load. It would be great if you could guide me about it.

Report saved to "ieee39_full_out.txt" in 0.0079 seconds.
GENROU (xl <= xd2) out of typical upper limit.

   idx     | values | limit
-----------+--------+------
 GENROU_1  | 0.012  | 0.001
 GENROU_2  | 0.042  | 0.036
 GENROU_3  | 0.036  | 0.003
 GENROU_4  | 0.025  | 0.001
 GENROU_5  | 0.050  | 0.001
 GENROU_7  | 0.031  | 0.002
 GENROU_8  | 0.029  | 0.006
 GENROU_9  | 0.018  | 0.001
 GENROU_10 | 0.003  | 0.000


Suspect initialization issue! Simulation may crash!

        Name          | Var. Value | Eqn. Mismatch
----------------------+------------+--------------
 LL_y TGOV1N TGOV1 10 | 17.879     | -5.290       


Initialization for dynamics completed in 0.0381 seconds.
Initialization failed!!
If you are developing a new model, check the initialization with
   andes -v 10 run -r tds --init /Users/mypc/PycharmProjects/andes_power_system_tool_project/andes_github/andes/cases/ieee39/ieee39_full.xlsx

**Reply by m-jam20** [reply]
My goal is to have different load levels set to  90%, 100%, 110%, and 120% of its base value. also to maintain the power balance, the output power of the generators should be correspondingly adjusted. Which parameters of the Generators should be modifies for this?

**Comment by jinningwang** [comment]
The IEEE 39-bus dynamic case is synthetic from experience. Feel free to modify the parameters when necessary.

Regarding the generator or load setting points, I believe there have been some discussions. Check discussion compass for more details.
